README.md

```diff
@@ -16,14 +16,107 @@ Concordia University
 
 ## Installation
 
-Firstly, you'll need an `.env` file, the project includes an `.env.example` file for reference. 
+### Windows Subsystem for Linux (WSL)
+This project runs best in WSL, so if you're on windows, you'll want to enable this feature. You can find more comprehensive 
+instructions for doing that [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10). But for completeness, here is a summary:
+
+**Note:** Make sure windows is up to date before starting this. You may also need to enable Virtualization in your BIOS
+
+1. Open a power shell window and run `dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart`
+2. in the same power shell  window run `dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart`, Then restart your machine.
+3. Download the latest [WSL2 updates here](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi), and install them.
+4. Back in the power shell run `wsl --set-default-version 2`
+5. Open the Microsoft store, then [navigate to the Ubuntu 20.04 page](https://www.microsoft.com/store/apps/9n6svws3rx71), and click "get".
+6. Once installed, launch the Ubuntu 20.04 App from the start menu and follow the account set-up instructions. Make sure to pick a memorable sudo password. 
+
+After completing the above steps, you should now have an Ubuntu terminal that you can run the rest of the installation in.
+
+### Brew (MacOS only)
+MacOS lacks a native package manager. If you haven't already you'll want to install [brew](https://brew.sh/).
+
+To install brew, in a terminal, simply run:
+```shell
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Once it completes, you can continue with the installation instructions.
+
+### PHP
+First you'll need PHP running on your command line. To do this install it, as well as some of its dependancies.
+
+For MacOS run: 
+```shell
+brew install php@7.2
```

Comment:
i had issue with composer when php is < @7.2, is it recommended to download @^8.0?

Reply:
Ah yeah you're right, that's a typo, good catch, fixed!